### ISSUE TITLE
Exception thrown when tiring to extend hit time

### ISSUE THREAD

**Issue by Ovadia-B** [aws/aws-sdk-java#1817]
I am extending hit expiration with UpdateExpirationForHIT while most of the time the requests are successful there are some requests that are failing. 

I tried to log out the time of expireAt value and it is correct. 

Extending expiration for HIT 3TRB893CSJTWCHXTCX7HTJTD8VJG7X, 
Now: Thu Nov 22 10:19:33 UTC 2018, 
Expire At: Thu Nov 22 11:19:33 UTC 2018

There is one hour between the time now and the time of expiry, but the service throws this exception:

com.amazonaws.services.mturk.model.AmazonMTurkException: The value 11 is invalid for ExpirationIncrementInSeconds. Valid values range from 60 to 31536000. (1542881973916) (Service: AmazonMTurk; Status Code: 400; Error Code: ParameterValidationError; Request ID: af1802ad-3ebf-4e59-9284-859f875db476)

**Comment by debora-ito** [comment]
Can you provide sample code that reproduces the problem?

**Comment by Ovadia-B** [comment]
Sure, here is the relevant code snippet:

`
        Date expireAt = new Date(Instant.now().plus(reward.getLifetime(), ChronoUnit.SECONDS).getEpochSecond() * 1000);
        logger.info("Extending expiration for HIT {}, Now: {}, Expire At: {}",
                hitId,
                new Date(Instant.now().getEpochSecond() * 1000),
                expireAt);

        mturk.updateExpirationForHIT(new UpdateExpirationForHITRequest()
                .withHITId(hitId)
                .withExpireAt(expireAt));`

**Comment by debora-ito** [comment]
I was able to reproduce it when the new expiration date is before the current expiration date - in this case, I got the message `The value -2752 is invalid for ExpirationIncrementInSeconds`. Apparently, the `updateExpirationForHIT()` only works with increments of the current expiration date, with the minimum of 60 seconds.

Is it possible that's the case?
In your example, the new expiration date is only 11 seconds after the current expiration date.

I didn't find anything in the Mechanical Turk API documentation mentioning this rule, it only says that if the expiration date is in the past the HIT will be immediately expired. So I'll go ahead and open an internal ticket asking the Mechanical Turk team to address the issue.
